test/plugins/individual-collateral/morpho-aave/MorphoAaveV2TokenisedDeposit.test.ts

```diff
@@ -233,6 +235,28 @@ const execTestForToken = ({
       await closeTo(methods.balanceUnderlying(alice), fraction(100))
     })
 
+    it('Withdrawals larger than deposits', async () => {
```

Comment:
I just ran this test with the `updateIndexes()` calls commented out, and it passed.

Reply:
Hm, so the change makes no difference?